src/mbed_tools/build/_internal/templates/mbed_config.tmpl

```diff
@@ -66,6 +66,12 @@ set(MBED_TARGET_DEFINITIONS{% for component in components %}
 {%- endif %}
 {% if mbed_ram_size is defined %}
     MBED_RAM_SIZE={{ mbed_ram_size | to_hex }}
+{%- endif %}
+{% if mbed_ram1_start is defined %}
+    MBED_RAM1_START={{ mbed_ram1_start | to_hex }}
```

Comment:
Do we need to also support MBED_RAM_2..MBED_RAM_N? https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L443xC/TOOLCHAIN_ARM/stm32l443xc.sct#L50 seems to want MBED_RAM2_* at least.

Reply:
@rwalton-arm Maybe it depends on whether we ever define `mbed_ram2_start` in targets.json?

Searching Mbed CLI 1 scripts (in mbed-os/tools/), it only has `mbed_ram_start`, not even ram1. So I wonder how (or if at all) `MBED_RAM1_START` is set by Mbed CLI 1.